src/js/utils/css.js

```diff
@@ -17,6 +17,8 @@ define([
                 return;
             }
             var el = document.createElement('div');
+
+            // TODO: Add warning if any styles have !important, since they will be stripped out here.
```

Comment:
Wanted to avoid making any significant changes here, but could probably add logic to to convert object to string if any field has !important in it.